From 00571896cc776222f14d9f67627ba6fb54512bd7 Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Sun, 6 Mar 2022 10:59:47 +0200
Subject: [PATCH] makepkg.conf: set LTOFLAGS="-flto=auto"

For use with pacman 6.0.1-4 which backports support for LTOFLAGS in
order to speed up builds that use gcc as the compiler. This is less
useful when clang is used, though a hack would be to specify in the
PKGBUILD `LTOFLAGS="-flto=thin"` (outside of any function).
---
 makepkg-x86_64.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/makepkg-x86_64.conf b/makepkg-x86_64.conf
index 19d16f7..7725296 100644
--- a/makepkg-x86_64.conf
+++ b/makepkg-x86_64.conf
@@ -45,6 +45,7 @@ CFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions \
         -fstack-clash-protection -fcf-protection"
 CXXFLAGS="$CFLAGS -Wp,-D_GLIBCXX_ASSERTIONS"
 LDFLAGS="-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now"
+LTOFLAGS="-flto=auto"
 #RUSTFLAGS="-C opt-level=2"
 #-- Make Flags: change this for DistCC/SMP systems
 #MAKEFLAGS="-j2"
